Directory.Build.props

```diff
@@ -19,7 +19,7 @@
     <IncludeSourceRevisionInInformationalVersion>false</IncludeSourceRevisionInInformationalVersion>
 
     <!-- False if running on CI, will prevent copying of files to local folders -->
-    <IsDesktopBuild Condition="'$(IsDesktopBuild)' == ''">true</IsDesktopBuild>
+    <IsDesktopBuild>false</IsDesktopBuild>
```

Comment:
was this change meant to be committed?

Reply:
Oh good catch, no this absolutely wasn't meant to be staged (it was merely how I was testing locally) thank you